script/conversion/convert_mkdocs_to_yaml_roles

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+import os
+import re
+from pathlib import Path
+from typing import Any
+
+import click
+import yaml
+
+
+def parse_markdown_to_yaml(input_file: str, destination: Path) -> str:
```

Comment:
`destination` is not used, and the name of the function seems misleading. `extract_title` maybe?